src/styles/inherited-scss/components/filter-panel/_date-filter.scss

```diff
@@ -21,7 +21,7 @@
   border-radius: 0 4px 4px 0;
   transition: color 0.3s ease-in-out, background 0.3s ease-in-out;
   background: $greyMiddle;
-  // background-color: aliceblue;  
+  // background-color: aliceblue;
```

Comment:
We’ll need to add `!*.scss` to `.prettierignore` and run `yarn fix:prettier` in a separate PR. This will reduce unexpected diffs like this (which is fine – no need to change anything).

Reply:
➡️ https://github.com/dtpstat/website/pull/74